agenda.html

```diff
@@ -0,0 +1,174 @@
+
+      </tr>
+      <tr>
+        <td class="time">16:10-17:00</td>
+        <td><span class="text-success font-weight-bold">TBA</span><br>HANNA NOWICKA, PhD Candidate<br>Oxford University
+        </td>
+      </tr>
+      <tr>
+        <td class="time">17:00-17:30</td>
+        <td class="font-weight-bold">Coffee Break</td>
+      </tr>
+      <tr>
+        <td class="time">17:30-18:20</td>
+        <td><span class="text-success font-weight-bold">TBA</span><br>ESTERA KOT, PhD Candidate<br>City & Warsaw
+          University of Technology</td>
+      </tr>
+      <tr>
+        <td class="time">18:20-18:30</td>
+        <td class="font-weight-bold">Closing Remarks</td>
+      </tr>
+    </table>
+  </section>
+  <!-- End Agenda Section -->
+  <!-- Start Abstracts Section -->
+  <section class="container bg-light">
+    <h3 class="text-center font-weight-bold">Abstracts</h3>
+    <div class="abstract">
+      <h4 class="text-success font-weight-bold">Practical transfer learning for NLP with spaCy and Prodigy</h4>
+      <p>Transfer learning has been called "NLP's ImageNet moment". Recent work has shown that models can be initialized
+        with detailed,
+        contextualised linguistic knowledge, drawn from huge samples of data. In this talk, I'll explain spaCy's new
+        support for efficient and easy transfer learning.
+      </p>
+    </div>
+    <div class="abstract">
+      <h4 class="text-success font-weight-bold">Deep understanding text-based models</h4>
+      <p>A data scientist should not only code machine learning models, but also understand their internal mechanisms.
+        How do we deal with problems with text modeling in business applications and research and development projects?
+        I will show several use cases that will include detection of emotions, language abuse and others. I will also
+        show a tool to explain suspicious results in data sets and model results.
+
+      </p>
+    </div>
+    <div class="abstract">
+      <h4 class="text-success font-weight-bold">Simple is not easy - the overview of two projects</h4>
+      <p>Even though the world of data science is fascinated by LSTMs and GANs, there are many commercial projects that
+        do not have to use such heavy artillery to provide value for the client. In my talk,
+        I will present two projects which main purpose was to create a tool for data analysis and visualization. First
+        is an application for constant audit and the second is Twitter explorer. Both are data-based web apps and and
+        gave our costumers major savings,
+        with IQR (Interquartile Range) being the most advanced statistical concept used. While building those projects
+        we encountered architectural and technical challenges. I will present some of them with our road to solution.
+        Our tech stack was: Python-Flask, MariaDB, Elasticsearch, Vue, Boostrap.
+      </p>
+    </div>
+  </section>
+  <!-- End Abstracts Section -->
```

Comment:
I would create here another small section like  "Follow the event"
like: Be up to date with our news - follow event on facebook (link to event)